package/MDAnalysis/topology/DLPolyParser.py

```diff
@@ -24,7 +24,7 @@
 """DL Poly format Topology Readers --- :mod:`MDAnalysis.topology.DLPolyParser`
 ==============================================================================
 
-Read DL Poly_ format topology files
+Read DL-POLY 4 and DL-POLY Classic format topology files
```

Comment:
What is the difference between DL-POLY, DL-POLY-4 and DL-POLY classic?

Just for clarity, because your edit seems to suggest DL_POLY == DL_POLY4

Reply:
I'll try to clarify. DL_POLY (with underscore and all capital letters) is the generic name. The latest release is: DL_POLY_4 v5.0.0 (https://www.scd.stfc.ac.uk/Pages/DL_POLY.aspx). This version is the descendant of DL_POLY 3. DL_POLY Classic is the direct descendant of DL_POLY 2 (https://www.ccp5.ac.uk/DL_POLY_CLASSIC). The main difference between DL_POLY 4 or 3 and DL_POLY Classic or 2 is the way parallelization is achieved.  "Static/equi-spatial Domain Decomposition (DD) mode" .vs."Replicated Data strategy".

I specified DL_POLY 4 because I could not find the manual of DL_POLY 3 for which  I don't know the format of the HISTORY file.